learning_wavelets/models/exact_recon_unet.py

```diff
@@ -170,14 +187,9 @@ def pad_power_of_two(x, n_layers):
 
     h = tf.shape(x)[1]
     w = tf.shape(x)[2]
-    
-    new_w = w
-    new_h = h
-    if(w % power_of_two != 0):
-        new_w = power_of_two - w % power_of_two + new_w
 
-    if(h % power_of_two != 0):
-        new_h = power_of_two - h % power_of_two + new_h
-    
+    new_w = tf.cast(power_of_two*tf.math.ceil(w/power_of_two), tf.int32)
+    new_h = tf.cast(power_of_two*tf.math.ceil(h/power_of_two), tf.int32)
```

Comment:
Excellent, can you change the documentation with this new approach (now it's not really related anymore)